go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -0,0 +1,43 @@
+package ledger
+
+import (
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	ledger "github.com/zondax/ledger-oasis-go"
+)
+
+var (
+	defaultDerivationPath = []uint32{44, 118, 0, 0, 0}
+)
+
+type Signer struct {
+	app *ledger.LedgerOasis
+}
+
+// NewSigner will attempt to initialize a Ledger Signer by running
+// FindLedger. This will connect to the first Ledger device found.
+func NewSigner() (signature.Signer, error) {
```

Comment:
Should this take a ledger device number and derivation path as an argument? The factory (see other comments) can then populate them with `0` (for now if the upstream module does not support multiple devices) and the derivation path for the `EntitySigner` role.

The derivation paths could be mapped similar to how the file signer maps filenames:
```go
	rolePEMFiles = map[signature.SignerRole]string{
		signature.SignerEntity:    FileEntityKey,
		signature.SignerNode:      FileIdentityKey,
		signature.SignerP2P:       FileP2PKey,
		signature.SignerConsensus: FileConsensusKey,
	}
```

Except that you would only support the `signature.SignerEntity` role for now (with the above derivation path).